src/components/RadioButton.js

```diff
@@ -0,0 +1,61 @@
+const React = require('react');
+
+const StyleConstants = require('../constants/Style');
+
+const RadioButton = React.createClass({
+  propTypes: {
+    activeButtonStyle: React.PropTypes.object,
+    buttonStyle: React.PropTypes.object,
+    checked: React.PropTypes.bool,
+    onClick: React.PropTypes.func,
+    style: React.PropTypes.object
+  },
```

Comment:
I just noticed `activeButtonStyle` that will override the active color but we're using `brandColor` elsewhere. Not sure what's preferred.